### PR TITLE
Optimise jenkins

### PR DESCRIPTION
**Pull request by amywalkerdev**
**Overall change:** Makes some changes to the Jenkins pipeline to try and speed up the average run time of CI and CD.

**Code changes:**

- Adds some parameters to the chromatic command so that the build carries on outside of the pipeline and doesn't wait for all stories to be checked. Chromatic is not a required check on Psammead currently so you can approve and merge without waiting for it to finish at all if you wish..
- Removes the run build step from the Test make command (as it is already run after install on the previous stage)
- Removed the build step from the publish script (as it is already run after install on the previous stage)
- Creates a new test:ci script that doesn't run build before the unit tests (as it is already run after install on the previous stage)

---

- [x] I have assigned myself to this PR and the corresponding issues
- [ ] Automated jest tests added (for new features) or updated (for existing features)
- [ ] This PR requires manual testing

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -3,6 +3,7 @@
 <!-- prettier-ignore -->
 | Version | Description |
 |---------|-------------|
+| 3.0.5 | [PR#(https://github.com/bbc/psammead/pull/) Modifying Jenkins pipeline to improve running time |
```
Comment (karinathomasbbc) [inline CHANGELOG.md]:
```suggestion
| 3.0.5 | [PR#3842(https://github.com/bbc/psammead/pull/3842) Modifying Jenkins pipeline to improve running time |
```

Comment (amywalkerdev) [reply CHANGELOG.md]: I always forget those! Thanks

File: package.json [hunk 2]
```diff
@@ -23,12 +23,14 @@
...
     "postinstall": "npm run build",
-    "publish": "npm run build && node scripts/publish",
+    "publish": "node scripts/publish",
     "regeneratePackageLocks": "node scripts/regeneratePackageLocks",
     "storybook": "NODE_ENV=development start-storybook -p 8180 -s .storybook/static -c .storybook",
     "test": "npm run test:lint && npm run test:unit && lerna run test",
+    "test:ci": "npm run test:lint && npm run test:unitci && lerna run test",
     "test:lint": "eslint --ext .js,jsx,json ./packages ./scripts && stylelint 'packages/**/*.js' 'packages/**/*.jsx' 'scripts/**/*.js'",
     "test:unit": "npm run build && jest --verbose --coverage",
+    "test:unitci": "jest --verbose --coverage",
```
Comment (karinathomasbbc) [inline package.json]: Is this is wee bit easier to read?
```suggestion
    "test:unit:ci": "jest --verbose --coverage",
```

**Review by karinathomasbbc — APPROVED** [review]

**Review by simonsinclair — APPROVED** [review]
Nice one, Amy!

**Review by jroebu14 — APPROVED** [review]

**Pull request merged**